src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public static final int MAX_SIZE_RANDOM_NUMBER = 10;
+    public static final int FIGURE_COUNT = 5;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure(int number) {
+        int figureNumber = random.nextInt(FIGURE_COUNT);
+        if (Main.NUMBER_OF_FIGURES / 2 >= number) {
```

Comment:
Do not use constants from main class, your service classes must be isolated and reusable